frontend/__tests__/components/interactive-chat-box.test.tsx

```diff
@@ -131,4 +131,71 @@ describe("InteractiveChatBox", () => {
     await user.click(stopButton);
     expect(onStopMock).toHaveBeenCalledOnce();
   });
+
+  it("should clear text input after image paste", async () => {
+    const onSubmit = vi.fn();
+    const onStop = vi.fn();
+    const onChange = vi.fn();
+
+    render(
+      <InteractiveChatBox
+        onSubmit={onSubmit}
+        onStop={onStop}
+        onChange={onChange}
+        value="test message"
+      />
+    );
+
+    const textarea = screen.getByRole("textbox");
+    expect(textarea).toHaveValue("test message");
+
+    // Create a mock image file
+    const imageFile = new File(["dummy content"], "test.png", { type: "image/png" });
+    const clipboardEvent = new Event("paste", { bubbles: true }) as any;
+    clipboardEvent.clipboardData = {
+      files: [imageFile],
+      getData: () => "",
+    };
+
+    // Trigger paste event
+    fireEvent(textarea, clipboardEvent);
+
+    // The text input should be cleared
+    expect(textarea).toHaveValue("");
```

Comment:
```suggestion
```

Instead of directly asserting the textarea content, we can just keep the assertion that `onChange` has been with `""`, this will also indirectly test that the textarea content will be cleared (actually depending on the implementation of `onChange`). This will also make it easier to test the behaviour since we don't want to force the `ChatInput` to call `onChange` as done in below.